platform/mock-radio.ts

```diff
@@ -34,10 +34,17 @@ export class MockNetwork {
   ) {
     const key = address + ':' + port.toString();
     if (this.udpListeners.has(key)) {
-      this.udpListeners[key].push(listener);
+      const listeners = this.udpListeners.get(key);
```

Comment:
curious how this is related to packaging? can you add a note in the PR description?

Reply:
Yeah this is not related.  The `tslint` rules in the [sample](https://github.com/actions-on-google/smart-home-local) seem to be more strict than my `eslint` rules.  These null checks were needed to satisfy those rules.  I'll note this in the PR description.

Reply:
Maybe I don't understand why tslint is throwing an issue. If `this.updListeners.has` resolves to true, then the getter should definitely be true. You can use a non-null assertion `!` at the end of a statement to guarantee it is not-null.

Reply:
It was complaining the value could be undefined.  Maybe you can set a map value as undefined in typescript.  I'll try with `!`

Reply:
`!` worked